Haulage/MainPage.xaml

```diff
@@ -1,35 +1,36 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             x:Class="Haulage.MainPage">
+             x:Class="Haulage.MainPage"
+             Shell.NavBarIsVisible="False">
```

Comment:
is this required? It doesn't like like there is any U.I elements it is setting to be invisible